modules/sdk/gradle-plugins-workspace/src/gradleTest/formatSourceJakarta/build.gradle

```diff
@@ -0,0 +1,18 @@
+task runGradleTest
+
+gradle.liferayWorkspace {
+	homeDir = new File(buildDir, "bundles")
+}
+
+runGradleTest {
+	dependsOn formatSourceJakarta
```

Comment:
It may also make sense to do a `doFirst` on the `formatSourceJakarta` task in order to assert the pre-transformed state (e.g., has `javax`, does not have `jakarta`).

Reply:
@drewbrokke this doesnt work since doFirst doesnt have priority on a depends on, we would have to manually execute the task within. Regardless I dont think this makes much sense from a testing perspective since we are sure it will always contain javax and not have jakarta since we were the ones that created the java content. The JS portlet gradleTest and all of the SF upgrade automation also went by this test logic